src/submodules/schedulers.jl

```diff
@@ -28,117 +28,108 @@ module Schedulers
 using Agents
 using Random: shuffle!, randsubseq
 
-export randomly, by_id, fastest, partially, by_property, by_type
+export AbstractScheduler,
+    Fastest,
+    ByID,
+    Randomly,
+    Partially,
+    ByProperty,
+    ByType
```

Comment:
All the old names need to be put in the deprecations! Also warnings need to be thrown in the ByType and ByProperty when used with a function!